src/label-for-moment/label-for-moment.test.js

```diff
@@ -0,0 +1,29 @@
+import labelForMoment from './label-for-moment';
+import moment from 'moment';
+import _ from 'lodash';
+
+describe('_.labelForMoment()', () => {
+  it('should return in minute if the difference time is less than 1 hour', () => {
+    expect(labelForMoment(moment().add(15, 'minute'))).to.equal('15m');
```

Comment:
just realized these are probably backward and should be subtract rather than add so it's 15m ago rather than in the future